modules/ui/sources/window.es

```diff
@@ -121,6 +121,9 @@ export default class {
 
     }.bind(this));
 
+    //mock default FF function
+    this.popup.enableOneOffSearches = function() {}
```

Comment:
You don't need to clobber the handler, there is a pref that disables one-off search buttons. See how it is done [here](https://github.com/mozilla/universal-search/blob/master/lib/universal-search.js#L56).